deployment/ansible/group_vars/packer

```diff
@@ -17,3 +17,5 @@ tiler_host: "tile.service.nyc-trees.internal"
 postgresql_username: nyctrees
 postgresql_password: nyctrees
 postgresql_database: nyc_trees
+
+tasseo_graphite_url: "http://$(curl -s http://instance-data.ec2.internal/latest/meta-data/public-hostname):8080"
```

Comment:
It's cool to use a hard-coded 8080 because we now use unique IP/hostnames to run multiple instances, rather than changing ports. Is my understanding correct?